src/client/services/gitHubService.ts

```diff
@@ -0,0 +1,26 @@
+import GitHub from 'github-api';
+
+export interface IGithubService {
+  /**
+   * Returns the gist of the latest commit for the given owner/repo.
+   */
+  getRepoLastCommitGist(owner: string, repo: string): Promise<{ message: string }>;
+}
+
+export class GitHubService implements IGithubService {
+  constructor(private gitHubApi: GitHub) {
+    console.log(this.gitHubApi);
+  }
+
+  public async getRepoLastCommitGist(owner: string, repo: string): Promise<{ message: string }> {
+    const repoData = await this.gitHubApi.getRepo(owner, repo);
```

Comment:
You should initialize the repoData in the constructor with orbs details ('orbs-network', 'orbs-network-go'), not in the social store. Doing so will change this class from being very general to orbs specific, and this is a good thing.
Also consider changing the class name from `GithubService` to `OrbsGithubService`.

Reply:
Changed the name

Reply:
And the initialization